src/components/Payment/PayPal/PayPal.tsx

```diff
@@ -1,11 +1,46 @@
+import { useState } from 'react';
```

Comment:
I would give more space between the papay not about redirection and the checkbox 
Eg: change msd__paypal--text margin to `margin: 20px 15px;`